persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala

```diff
@@ -115,6 +123,9 @@ class HmdaFileParser(submissionId: SubmissionId) extends HmdaPersistentActor wit
       log.debug(s"Parsing completed for $submissionId")
       publishEvent(ParsingCompleted(submissionId))
 
+    case CompleteParsingWithErrors =>
+      log.debug(s"Parsing completed for $submissionId, errors found")
```

Comment:
Ohh, I see. Can you see it [here](https://github.com/schbetsy/hmda-platform/blob/e53e31bace3d99143d1066a9c498b8a960b58b42/persistence/src/main/scala/hmda/persistence/processing/HmdaFileParser.scala#L129)? I think there might be a bug in GitHub, because I added that line, but sometimes it disappears in the PR Review view.